static/css/devhub/new-landing/sections/my-addons.less

```diff
@@ -1,3 +1,150 @@
 .DevHub-MyAddons-copy {
   max-width: 32em;
 }
+
+.DevHub-MyAddons-item {
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #e4e4e4;
+  padding-bottom: 25px;
+}
+
+.DevHub-MyAddons-item-modified {
+  margin-left: auto;
+  align-self: flex-end;
+}
+
+.DevHub-MyAddons-item-details {
+  margin-left: 25px;
+
+  .DevHub-MyAddons-item-name {
+    font-size: 24px;
+    font-weight: 400;
+  }
+
+  a {
+    margin-left: 5px;
+    font-size: 15px;
+    text-decoration: none;
+    color: @color-button-default;
+
+    &.DevHub-MyAddons-version-status-incomplete,
+    &.DevHub-MyAddons-version-status-deleted,
+    &.DevHub-MyAddons-version-status-rejected,
+    &.DevHub-MyAddons-version-status-disabled,
+    &.DevHub-MyAddons-version-status-none {
+      color: #9b9898;
+    }
+
+    &.DevHub-MyAddons-version-status-nominated {
+      color: #ff9216;
+    }
+
+    &.DevHub-MyAddons-version-status-approved {
+      color: #329902;
+    }
+
+    &.DevHub-MyAddons-version-status-admin-disabled {
+      color: #d92215;
+    }
+  }
+
+  .DevHub-MyAddons-item-channel-listed,
+  .DevHub-MyAddons-item-channel-unlisted {
+    border-radius: 4px;
+    padding: 2px 5px;
+    font-size: 13px;
+    font-weight: 500;
+    color: #fff;
+  }
+
+  .DevHub-MyAddons-item-channel-listed {
+    background-color: #01bdad;
+  }
+
+  .DevHub-MyAddons-item-channel-unlisted {
+    background-color: #7a2f7a;
+  }
+}
+
+.DevHub-MyAddons-item-icon {
+  width: 48px;
+  height: 48px;
+}
+
+.DevHub-MyAddons-item-versions {
+  display: flex;
+  justify-content: space-between;
+}
+
+.DevHub-MyAddons-item-version-details {
+  width: 320px;
+}
+
+.DevHub-MyAddons-item-buttons {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-start;
+
+  > a:first-child {
```

Comment:
Yeah, afair it matched the first a in `.DevHub-MyAddons-item-buttons-submit` too since we have something like `<div .DevHub-MyAddons-item-buttons> <a> <div .DevHub-MyAddons-item-buttons-submit> <a> <a> </div></div>``